bindings/netstandard/ElectionGuard/ElectionGuard.Decryption/Tally/CiphertextTallySelection.cs

```diff
@@ -119,10 +119,7 @@ protected ElGamalCiphertext Accumulate(ElGamalCiphertext ciphertext)
     {
         lock (_lock)
         {
-            var newValue = Ciphertext.Add(ciphertext);
-            var oldValue = Ciphertext;
-            Ciphertext = newValue;
-            oldValue.Dispose();
+            Ciphertext = Ciphertext.Add(ciphertext);
```

Comment:
i ran the memory check to validate that this doesn't produce a leak and i don't think it does but i'm not 100% sure